docs/linter/eslint-plugin/rules/o3r-categories-tags.md

```diff
@@ -0,0 +1,70 @@
+# @o3r/o3r-categories-tags
+
+Ensures that @o3rCategories and @o3rCategory are used with correct value.
+
+## How to use
+
+```json
+{
+  "@o3r/o3r-categories-tags": [
+    "error",
+    {
+      "supportedInterfaceNames": ["NestedConfiguration", "Configuration", "CustomConfigurationInterface"],
+      "globalConfigCategories": ["globalCategory"]
+    }
+  ]
+}
+```
+
+## Valid code example
+
+```typescript
+/**
+ * @o3rCategories localCategory
+ */
+export interface ConfigExample extends Configuration {
+  /**
+   * @o3rCategory globalCategory
+   */
+  prop1: string;
+  /**
+   * @o3rCategory localCategory
+   */
+  prop2: string;
+}
+```
+
+## Invalid code example
+
+```typescript
+export interface ConfigExample extends Configuration {
+  /**
+   * @o3rCategory unknownCategory
+   */
+  prop: string | number;
+}
+```
+
+```typescript
```

Comment:
why is this example invalid?

Reply:
Because you cannot redeclare a category

Reply:
It could be nice to explain why each example is incorrect to help users identify the issue